src/api/index.py

```diff
@@ -20,18 +20,72 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+from enum import Enum
 from typing import Optional
 
 from fastapi import FastAPI, HTTPException
 
 from src.classes.analyze_video_response import AnalyzeVideoResponse
-from src.services.image_analysis_service import run_image_analysis
-from src.services.video_service import download_video
+from src.classes.video_data import VideoData
+from src.classes.yolo_model import YoloModel
+from src.services.image_analysis_service import AnalyzerService
+from src.services.prep_service import DataPreparationService
+from src.utils import default_variables as dv
 from src.utils.default_variables import target_item, video_url
 
 app = FastAPI()
 
 
+class ModelChoices(str, Enum):
```

Comment:
Thanks for adding this `enum`! I noticed yesterday we had a lot of hard coded strings. 😅 Let's move this to a more central place, perhaps just to a new `src/enums/model_enums.py` file to start with. Then we can update all the hard coded strings to reference this. Not only in this file but in our other files that reference these (like `yolo_model.py`). Sound good?